src/utils/NetworkUtils.ts

```diff
@@ -60,10 +60,12 @@ export function chainIsOPStack(chainId: number): boolean {
   return [
```

Comment:
nb. this will treat Blast as a generic OP stack implementation. I _think_ that's OK, but there will definitely be places where we need to override OP stack defaults (i.e. finalization challenge window settings).